ui/raidboss/timeline.js

```diff
@@ -794,22 +794,67 @@ class TimelineController {
     this.dataFiles = {};
     // data files not sent yet.
     this.timelines = null;
+
+    // Fix for #1453
+    // Countdown state
```

Comment:
Instead of magic numbers can I suggest `this.countdownActive` and `this.wipedDuringCountdown`?

Reply:
Not sure if you're suggesting separating this into two separate flags or if you're suggesting there should be an enum for this?

Reply:
I think either is fine, whichever you think makes this more readable.  Mostly I would just prefer names instead of numbers.

Reply:
This has been adjusted with latest commit, but might not be quite to your liking yet so not marking this resolved yet.